packages/emoji/src/utils/__test__/shortnameToUnicode.test.ts

```diff
@@ -0,0 +1,10 @@
+import shortnameToUnicode from '../shortnameToUnicode';
+
+describe('shortnameToUnicode', () => {
+  it('executes in less than 100ms', () => {
```

Comment:
I don't like the idea of testing the performance of a function in a unit test. On a very slow computer this test may fail, which is wrong.

It would make more sense to test only if the function works correctly.

Reply:
added some tests...